tools/PublishModules.ps1

```diff
@@ -174,10 +179,20 @@ function Change-RMModule
 		  Write-Output "Expanding $zipPath"
 		  Expand-Archive $zipPath
 		  Write-Output "Adding PSM1 dependency to $unzippedManifest"
-		  Add-PSM1Dependency -Path $unzippedManifest
-		  Write-Output "Removing module manifest dependencies for $unzippedManifest"
-		  Remove-ModuleDependencies -Path $unzippedManifest
 
+          if ($moduleName -ne "AzureStack")
```

Comment:
I don't think this is the right way to do this.  This function shouldn't be called on the AzureStack module, so that is where the change should be made